app/src/main/AndroidManifest.xml

```diff
@@ -38,7 +38,14 @@
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
             android:hardwareAccelerated="true"
             android:windowSoftInputMode="adjustResize"
-            />
+            android:exported="true"
+            >
+            <intent-filter>
+               <action android:name="android.intent.action.SEND" />
+               <category android:name="android.intent.category.DEFAULT" />
+               <data android:mimeType="text/*" />
```

Comment:
Can you please remove the mimetype.

Reply:
[a91d72b](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/22/commits/a91d72b4f0fa47390729212bd4c4f3fe2b828a37)